vllm/v1/core/single_type_kv_cache_manager.py

```diff
@@ -511,40 +530,24 @@ def find_longest_cache_hit(
                 break
         return computed_blocks
 
-    def remove_skipped_blocks(self, request_id: str, num_computed_tokens: int) -> None:
-        # Remove the blocks that are no longer be in the chunked attention
-        # window and skipped during the attention computation.
-
-        # [chunk 0][chunk 1]local_attention_start_idx ... current
-        # we computed previous number of chunks to get the idx of
-        # current chunk window starting offset,
-        # e.g. for computed 1024 tokens, the 1024th token (0 indexed)
-        # is in the second chunk, there are 1 prev chunk, the start idx
-        # is 1024. for 1023, it will be 0.
-        num_cached_block = self.num_cached_block.get(request_id, 0)
+    def get_num_skipped_tokens(self, num_computed_tokens: int) -> int:
+        """
+        Get the last token (leftmost token) index that is inside chunked local
+        attention window.
+
+        Args:
+            num_computed_tokens: The number of tokens that have been computed.
+
+        Returns:
+            The last token (leftmost token) index that is inside chunked local
+            attention window.
+        """
         local_attention_start_idx = (
             (num_computed_tokens)
             // self.attention_chunk_size
             * self.attention_chunk_size
         )
```

Comment:
```suggestion
num_skipped_chunks = num_computed_tokens // self.attention_chunk_size
return num_skipped_chunks * self.attention_chunk_size
```

at least can you change `local_attention_start_idx` to some variable name related to num_skipped_tokens?

Reply:
Updated.